cmake/linux/installer

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -e
+[ "$1" ]
+
+VERSION="$1"
+DIR="$HOME"/.lmms-$VERSION
```

Comment:
We can, but `.deb` files don't come bundled with dependencies and generally require a package manager to fulfill them which varies between distros so we'll likely find ourselves offering various debs for different Debian-based distros.  I'd assume OpenSUSE/SUSE and Fedora/RHEL/Centos suffer the same problems. I believe this is part of the problem people have with the KXStudio repos (work, but bundle some libraries people already have) although I don't know the specifics, I've just seen people on IRC complaining about the DEBs from there.

I've supported desktop software on Linux before and the `.run` files work very well for what we're doing.  They're essentially acting just like the `dmg` and `exe` files do, but for Linux.